CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# [Unreleased](https://github.com/ImperialCollegeLondon/virtual_rainforest/)
```

Comment:
I'm not sure why you've added an empty changelog here...? Maybe you guys do want to have a discussion about having a changelog at some point but:

1. Adding it should really be a separate PR (or at least a separate git commit!)
2. I'd wait until you've got something to put in it
3. I'd open raise it as an issue first

Reply:
It seemed to be connected to the recommended template, that was my only motivation. I'll remove it.